src/structstore_pybind.cpp

```diff
@@ -162,6 +176,9 @@ void register_structstore_methods(py::class_<T>& cls) {
         str << store;
         return str.str();
     });
+    cls.def("__eq__", [](T& t, T& o) {
+        return static_cast<StructStore&>(t) == static_cast<StructStore&>(o);
+    });
```

Comment:
Here I'm not clearly decided yet... I guess if just the pointer addresses are compared, this should be made obvious at this place? When calling the comparison operator on a structured object, I would expect a recursive comparison instead